### PR TITLE
Add autocomplete, tabindex, and aria attributes to the honeypot field

### PR DESCRIPTION
**Pull request by junowilderness**
This merge provides to accessibility/UX improvements:

- Add [attribute tabindex="-1"](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex) to the honeypot field so that users navigating with the tab key won't stop on the field.

> A negative value (usually tabindex="-1") means that the element is not reachable via sequential keyboard navigation, but could be focused with Javascript or visually by clicking with the mouse.

- Add [attribute autocomplete="off"](https://developer.mozilla.org/en-US/docs/Web/Security/Securing_your_site/Turning_off_form_autocompletion) to the honeypot form to prevent browsers from auto-filling the field.

> As website author, you might prefer that the browser not remember the values for such fields, even if the browser's autocomplete feature is enabled.

- Set the ARIA labels necessary for assistive technologies to recognize the field and to ignore it.

### CODE REVIEW

**Review by maelanleborgne — APPROVED** [review]

**Comment by isometriks** [comment]
@maelanleborgne do you have merge permissions? You can feel free to merge these if you think it's okay, I don't really have time to maintain this.

**Comment by maelanleborgne** [comment]
> @maelanleborgne do you have merge permissions? You can feel free to merge these if you think it's okay, I don't really have time to maintain this.

Yes I will merge these once the CI is running.

**Comment by cilefen** [comment]
@maelanleborgne I rebased this back to 1.2.0 so I can use it, which causes conflicts with master. The master branch has many changes. If you are thinking of merging some pulls and releasing let me know and I will rebase this on to master.

**Comment by isometriks** [comment]
Yeah, rebase on master, I'm not gonna do a release for older versions of Symfony unless you really need this and can't upgrade, I'll just merge it onto the branch you want and do an older release, but I'll still want this on master too

**Comment by cilefen** [comment]
@isometriks It is rebased on master.

**Comment by siwa-ddiesenreither** [comment]
@isometriks are there any plans of merging this? thank you :)

**Review by maelanleborgne — APPROVED** [review]
Looks good to me, thank you @junowilderness

**Pull request merged**
